jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP_OBS

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Lets call this job:
`JGLOBAL_PREP_OCEAN_OBS` to be consistent with how jobs are named.
This job will be called for both `gdas` and `gfs`

Reply:
done